identity/registry/transactor.go

```diff
@@ -444,16 +445,39 @@ type SettleWithBeneficiaryRequest struct {
 	Registry    string   `json:"registry"`
 }
 
+// QueueResponse represents the queue response from transactor.
+type QueueResponse struct {
+	ID    string `json:"id"`
+	Hash  string `json:"tx_hash"`
+	State string `json:"state"`
+	Error string `json:"error"`
+}
+
+// GetQueueStatus returns the queued transaction status from transactor.
+func (t *Transactor) GetQueueStatus(ID string) (QueueResponse, error) {
```

Comment:
This is not used or am I not finding references?

Reply:
ur just not finding the ref probably due to interface.